recipes/diffpy.labpdfproc/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{%- set name = "diffpy.labpdfproc" -%}
+{%- set version = "0.1.2" -%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/d/diffpy_labpdfproc/diffpy_labpdfproc-{{ version }}.tar.gz
+  sha256: 521dd9a24c062ca52d69ef5e571c8551dbff04a23bad55bb8a7cf5940eba9bf6
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - labpdfproc = diffpy.labpdfproc.labpdfprocapp:main
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools
+    - setuptools-git-versioning >=2.0
+    - pip
+
+  run:
+    - python >=3.10
+    - numpy
+    - diffpy.utils
+    - pandas
+    - scipy
+
+test: 
```

Comment:
Could you add a command line test for `labpdfproc`? See:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
Fixed:

```
  imports:
    - labpdfproc
```

Reply:
Unfortunately, the test failed using the above. It could be due to the namespace package structure  since it is structured as `labpdfproc = diffpy.labpdfproc.labpdfprocapp:main`

```
  imports:
    - diffpy.labpdfproc
```